### PR TITLE
fix/ECMS-7913: fix css style for editing toolbar

### PR DESCRIPTION
**Pull request by aelbenney**
The editing toolbar has an `absolute position` that makes it hide the text in the edit area. So we need to adjust it than modify her position attribute as relative .

### CODE REVIEW

File: platform-ui-skin/src/main/webapp/skin/less/ecms/skin/ecms-resources-wcmskin.less [hunk 1]
```diff
@@ -209,30 +210,33 @@
...
 	right: 0;
 	top: 0;
 	width: 160px;
-	z-index: 1;
 	.edittingToolBar {
-		background: @primaryLightColor;
+		background: #578ecc;
 		padding: 5px;
 		.border-radius(3px 3px 0px 0px);
 	    position: absolute;
```
Comment (thomasdelhomenie) [inline platform-ui-skin/src/main/webapp/skin/less/ecms/skin/ecms-resources-wcmskin.less]: When the portlet toolbar is displayed, it works fine, but when the portlet toolbar is hidden, the blue buttons do not appear. It seems to be relative to this position absolute.

File: platform-ui-skin/src/main/webapp/skin/less/ecms/skin/ecms-resources-wcmskin.less [hunk 2]
```diff
@@ -209,30 +210,37 @@
 	}
 }
 
+/******** middleDecorator ***************/
+.middleDecorator{
+    position: relative;
+}
```
Comment (thomasdelhomenie) [inline platform-ui-skin/src/main/webapp/skin/less/ecms/skin/ecms-resources-wcmskin.less]: Is this change really necessary ? It seems it is not used in CLV portlet.

File: platform-ui-skin/src/main/webapp/skin/less/ecms/skin/ecms-resources-wcmskin.less [hunk 3]
```diff
@@ -209,30 +210,37 @@
...
 	float: right;
-	position: relative;
 	right: 0;
+	position: unset;
 	top: 0;
 	width: 160px;
-	z-index: 1;
 	.edittingToolBar {
-		background: @primaryLightColor;
+		background: #578ecc;
```
Comment (thomasdelhomenie) [inline platform-ui-skin/src/main/webapp/skin/less/ecms/skin/ecms-resources-wcmskin.less]: Shouldn't we keep @primaryBackground to be consistent ?

File: platform-ui-skin/src/main/webapp/skin/less/ecms/skin/ecms-resources-wcmskin.less [hunk 4]
```diff
@@ -209,30 +210,37 @@
...
+	    top: -40px;
 
 		.btn-group {
 			position: static;
 			.btn {
 				padding: 3px 5px;
-				border: 1px solid @primaryDarkColor;
-				background: @primaryLightColor;
+				border: 1px solid #446cc6;
+				background: #3b7bc2;
```
Comment (thomasdelhomenie) [inline platform-ui-skin/src/main/webapp/skin/less/ecms/skin/ecms-resources-wcmskin.less]: Same remarks here.

File: platform-ui-skin/src/main/webapp/skin/less/ecms/skin/ecms-resources-wcmskin.less [hunk 5]
```diff
@@ -209,30 +210,37 @@
...
+/******** middleDecorator ***************/
+.middleDecorator{
+    position: relative;
+}
+
 /******** EdittingToolBar **************/
 
+.UIContainer  {
+	position: relative;
+}
```
[the remark this answers is not in the document]
Comment (SaraBoutej) [reply platform-ui-skin/src/main/webapp/skin/less/ecms/skin/ecms-resources-wcmskin.less]: I tried to put the "position: relative" in the global container of the contents to make the editing toolbar buttons clearly visible when the portlet header is displayed or hidden, there are two other classes (. NormalContainerBlock. UIComponentBlock, .VIEW-CONTAINER. VIEW-BLOCK) that I can use instead of the class (UIContainer)
--------------------------------------------------------
<div class="UIContainer " id="OfficeMiddleCLVContainer">

	<div class="NormalContainerBlock UIComponentBlock">
		
			<div class="VIEW-CONTAINER VIEW-BLOCK">
...
---------------------------------------------------------------
But i don't think that using ( position: relative ) in the class UIContainer can bring side effect in the platform, it only makes the positions of the childs relative to UIContainer! WDYT?

[the remark this answers is not in the document]
Comment (SaraBoutej) [reply platform-ui-skin/src/main/webapp/skin/less/ecms/skin/ecms-resources-wcmskin.less]: I tried to put the "position: relative" in the global container of the contents to make the editing toolbar buttons clearly visible when the portlet header is displayed or hidden, there are two other classes (. NormalContainerBlock. UIComponentBlock, .VIEW-CONTAINER. VIEW-BLOCK) that I can use instead of the class (UIContainer)
But i don't think that using ( position: relative ) in the class UIContainer can bring side effect in the platform, it only makes the positions of the childs relative to UIContainer!
`<div class="UIContainer " id="OfficeMiddleCLVContainer">

	<div class="NormalContainerBlock UIComponentBlock">
		
			<div class="VIEW-CONTAINER VIEW-BLOCK">..`

[the remark this answers is not in the document]
Comment (thomasdelhomenie) [reply platform-ui-skin/src/main/webapp/skin/less/ecms/skin/ecms-resources-wcmskin.less]: Can't we just scope a bit more the property to make sure there is no side effect, for example by using ".UICLVPortlet .UIContainer" instead ?
